cmd/server/main.go

```diff
@@ -0,0 +1,7 @@
+package server
+
+import "log"
+
+func StartServer() {
```

Comment:
**Suggestion:**
As a convention, I would like to have just a `main` function for each `main.go` of each process inside of `cmd`

Reply:
How works this?, if I'm add a main function in this server package this said that "main is unused".

![image](https://user-images.githubusercontent.com/48216252/185499965-7d0801e2-f126-455c-b435-35e1e035b135.png)

Reply:
![image](https://user-images.githubusercontent.com/48216252/185500128-f77207e3-1c10-49b9-a9cf-6ce8bc56c04b.png)